app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -76,29 +76,36 @@ protected function indexData($request)
         );
     }
 
-    protected function getSubbrowserItems($digitalPublicationId)
+    protected function getBrowserData($prependScope = [])
     {
-        $articles = $this->repository->where('digital_publication_id', $digitalPublicationId)
-                                     ->withDepth()
-                                     ->defaultOrder()
-                                     ->get();
+        $query = $this->repository->withDepth()->defaultOrder();
+
+        $search = $this->request->get('search', $prependScope['search'] ?? null);
+        if ($search) {
+            $query->whereRaw('LOWER(title) LIKE ?', ['%' . strtolower($search) . '%']);
```

Comment:
TIL that MySQL doesn't have the `ILIKE` operator like Postgres does.

Reply:
I looked that up, and apparently people found the DB performance of ILIKE is super-heavy. So the recommendation is to use LOWER().